src/components/colors/Colors.stories.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import { colors } from './Colors';
+
+const ColorPageContainer = styled.div`
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr 1fr;
+  font: HK Grotesk;
+`;
+
+const ColorContainer = styled.div`
+  height: 40px;
+  border-radius: 5px;
+  border: 1px solid black;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font: bold;
+  margin: 20px 7px;
+
+
+  ${(p) => p.white && "color: white"}
+  ${(p) => p.empty && "border: 1px dotted grey;"}
+`;
+
+// More on default export: https://storybook.js.org/docs/react/writing-stories/introduction#default-export
+export default {
+  title: 'Color Palette',
+  component: colors,
+};
+
+let colorVisual = Array.from(Object.keys(colors));
+
+colorVisual.splice(-9, 0, 'empty');
+colorVisual.splice(-9, 0, 'empty');
+colorVisual.splice(-5, 0, 'empty');
+colorVisual.splice(-3, 0, 'empty');
+colorVisual.splice(-3, 0, 'empty');
+colorVisual.splice(-3, 0, 'empty');
```

Comment:
Just for my personal understanding, what does this do?

Reply:
haha it's to keep the color layout we have on Aurora spec! Keeping the some family of colors in one row. Plus I made the empty spots look cool :p

PS: `empty` is rendered as the grey blank divs